static/app/components/datePageFilter.tsx

```diff
@@ -53,10 +53,12 @@ function DatePageFilter({router, resetParamsOnChange, ...props}: Props) {
 
     return (
       <PageFilterDropdownButton
+        {...getActorProps()}
```

Comment:
Why did you have to move `getActorProps` up

Reply:
I thought that it overrode `hideBottomBorder={false}`, but just looked at it again and doesn't look like that's the case.